sonar-project.properties

```diff
@@ -38,7 +38,7 @@ sonar.exclusions=\
   **/mock_responses/**/*, \
   **/mocks/**/*, \
   **/node_modules/**/*, \
-  **/public/**/*, \
+  **/target/**/*, \
   **/scripts/**/*, \
```

Comment:
Not directly related, but `**/scriots/**/*` is a bit too broad. It excludes valid source directories such as `src/plugins/data/server/scripts`

Reply:
I attempted to enumerate the script directories that I don't care about in https://github.com/elastic/kibana/pull/190347/files. I can't guarantee it's correct, so use at your own risk 🙈

Reply:
I went with https://github.com/elastic/kibana/pull/190350/commits/09cbbc3f1071290f74ee41b80ae304bce89d2836 in case new folders are added.  I didn't include the root level `scripts` directory because it's not included initially.  Seem reasonable?

Reply:
> I went with [09cbbc3](https://github.com/elastic/kibana/commit/09cbbc3f1071290f74ee41b80ae304bce89d2836) in case new folders are added. I didn't include the root level `scripts` directory because it's not included initially. Seem reasonable?

I'm surprised that `**/scripts/**/*` wouldn't match the root level `scripts`, but if that's how it behaved before, then what you have seems fine to me.

Reply:
> > I went with [09cbbc3](https://github.com/elastic/kibana/commit/09cbbc3f1071290f74ee41b80ae304bce89d2836) in case new folders are added. I didn't include the root level `scripts` directory because it's not included initially. Seem reasonable?
> 
> I'm surprised that `**/scripts/**/*` wouldn't match the root level `scripts`, but if that's how it behaved before, then what you have seems fine to me.

To clarify, it would match it but we're not including it in folder sources list anyways https://github.com/elastic/kibana/blob/bff550081f54e7ab016eaac15fa3c7d1bcd12097/sonar-project.properties#L5-L9

Reply:
Ah my mistake. LGTM!